mpc_ros1_rbt4_node:
  N: 30
  dt: 0.1
  v_max: 0.3
  v_min: 0.0
  ref_topic_name: '/UGV4/ref_trajectory'
  pose_topic_name: '/UGV4/odom_ground_truth'
  cmd_topic_name: '/UGV4/jackal_velocity_controller/cmd_vel'
  pre_topic_name: '/UGV4/pre_path'
  frame_id: "map"
